Refactor 'Tag To Category' Metabox into new flow

anandB000 opened the pull request:
Fix #41

anandB000 committed:
Refactor 'Tag To Category' Metabox into new flow

Fix #41

sudar commented on include/Metabox/Posts/Tagtocategory.php:
```diff
@@ -0,0 +1,110 @@
+<?php
```
The filename should be `TagToCategory`. Class name should be `BM_Metabox_Posts_TagToCategory`

anandB000 replied:
Fixed

sudar commented on include/Metabox/Posts/Tagtocategory.php:
```diff
@@ -0,0 +1,110 @@
+
+    protected function initialize() {
+        $this->meta_box_slug         = 'bm-posts-tag-to-category';
+        $this->messages['box_label'] = __( 'Move Posts Tag To Category', 'bulk-move' );
```
Change label to `Move Posts from Tag to Category`

anandB000 replied:
Fixed

sudar commented on include/Metabox/Posts/Tagtocategory.php:
```diff
@@ -0,0 +1,110 @@
+    protected function initialize() {
+        $this->meta_box_slug         = 'bm-posts-tag-to-category';
+        $this->messages['box_label'] = __( 'Move Posts Tag To Category', 'bulk-move' );
+        $this->action                = 'move_tagtocategory';
```
This should be `move_tag_to_category`

anandB000 replied:
Fixed

sudar commented on include/Metabox/Posts/Tagtocategory.php:
```diff
@@ -0,0 +1,110 @@
+
+            </table>
+            <p>
+                <?php _e( 'If the post contains other tags, then', 'bulk-move' ); ?>
```
This should be `If the post contains other categories`

sudar commented on include/Metabox/Posts/Tagtocategory.php:
```diff
@@ -0,0 +1,110 @@
+
+            if ( $options['overwrite'] ) {
+                // Override is set, so remove all common tags.
+                $current_tags = array();
```
If overwrite is set, then the categories should be reset, not tags.

sudar commented on include/BulkMove.php:
```diff
@@ -263,6 +263,7 @@ private function get_post_admin_page() {
 		// TODO: Add other metaboxes.
 		$posts_page->add_metabox( new BM_Metabox_Posts_Category() );
 		$posts_page->add_metabox( new BM_Metabox_Posts_Tag() );
+        $posts_page->add_metabox( new BM_Metabox_Posts_Tagtocategory() );
```
Alignment is wrong from this line also.

anandB000 committed:
The filename should be TagToCategory.

Filename changed to 'TagToCategory' and also class name changed to 'BM_Metabox_Posts_TagToCategory'

see #41

anandB000 committed:
Label changed to 'Move Posts from Tag to Category'

see #41

anandB000 committed:
Change action 'move_tag_to_category'

see #41

anandB000 committed:
Change label to ''If the post contains other categories'

see #41

anandB000 committed:
Update tagtocategory 'move' code

see #41
